io/io/src/TBufferFile.cxx

```diff
@@ -658,8 +675,13 @@ void TBufferFile::WriteFloat16(Float_t *f, TStreamerElement *ele)
 ///  see example of use of the Double32_t data type in tutorial double32.C
 ///  \image html tbufferfile_double32.gif
 
-void TBufferFile::WriteDouble32(Double_t *d, TStreamerElement *ele)
+void TBufferFile::WriteDouble32(Double_t *d, TStreamerElement *elem)
 {
+   // The parameter should be const, however we have not yet decided how to
+   // transition the signature since the function is virtual.  This insures
+   // that the function does not inadvertently use non-const part of
```

Comment:
```suggestion
   // transition the signature since the function is virtual. This ensures
   // that the function does not inadvertently use non-const parts of
```